### ISSUE TITLE
Check connection and authorizations to DB before starting migration

### ISSUE THREAD

**Issue by poignanj** [PasserineBird/sparrow-tools#2]
- [x] Check MySQL connections
- [ ] Check MySQL user authorizations
- [x] Check MongoDB connections
- [ ] Check MongoDB authorizations

**Comment by poignanj** [comment]
Not checking authorizations for now. Postponed to later amelioration
